gaugeprogressview/src/main/java/com/zigis/gaugeprogressview/GaugeProgressView.kt

```diff
@@ -272,7 +274,11 @@ open class GaugeProgressView : View {
 
     fun setProgress(newProgress: Int) {
```

Comment:
better supply animated parameter to this method with a default value true.

Reply:
Done